crates/rattler_index/Cargo.toml

```diff
@@ -3,22 +3,49 @@ name = "rattler_index"
 version = "0.20.12"
 edition.workspace = true
 authors = []
-description = "A crate that indexes directories containing conda packages to create local conda channels"
+description = "A crate to index conda channels and create a repodata.json file."
 categories.workspace = true
 homepage.workspace = true
 repository.workspace = true
 license.workspace = true
 readme.workspace = true
+default-run = "rattler-index"
+
+[[bin]]
+name = "rattler-index"
+path = "src/main.rs"
```

Comment:
i would suggest we don't publish binaries for rattler-index in this repo as this will probably complicate the release-plz workflow in this repo quite a bit. instead, we can only build binaries on conda-forge (and other package managers like brew), imo this should be enough